PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -342,6 +327,27 @@ public static void checkGoogleMobileAdsCompatibility(@NonNull String googleAdsVe
         }
     }
 
+    /**
+     * Sets full valid URL for the /status endpoint of the PBS.
+     * Request to /status is sent when you call {@link PrebidMobile#initializeSdk(Context, SdkInitializationListener)}.
+     *
+     * @see <a href="https://docs.prebid.org/prebid-server/endpoints/pbs-endpoint-status.html">GET /status</a>
+     */
+    public static void setCustomStatusEndpoint(String url) {
+        if (url != null) {
```

Comment:
nitpic: I always sugest to use 
- positive logic in the `if` statmets
- make a code as plain as possible
- make an early return wherever it makes sense 

so the following statement is more cleaner 

``` 
void foo(String url) {
    // Process precondition
    if (url == null) {
        return 
    }

    ...

    // Run the function body
}
````